addons/resource_remaps/resource_remaps_plugin.gd

```diff
@@ -23,7 +23,6 @@
 @tool
 extends EditorPlugin
 
-const CONTROL_CONTAINER = CONTAINER_PROJECT_SETTING_TAB_RIGHT
```

Comment:
I'd prefer this approach instead:

`const CONTROL_CONTAINER: CustomControlContainer = CONTAINER_PROJECT_SETTING_TAB_RIGHT`

Reply:
I've updated the PR accordingly.